spark/components/card/react/SprkCard.stories.js

```diff
@@ -21,7 +23,9 @@ export const standout = () => (
     additionalContentClasses="
           sprk-o-Stack
           sprk-o-Stack--medium"
-  />
+  >
+    Base Card Content
```

Comment:
Should be "Standout Card Content" or lorem ipsum or something.

Reply:
Good catch!

Reply:
done!